sql/testdata/drop_table

```diff
@@ -0,0 +1,30 @@
+statement ok
+CREATE TABLE a (id INT PRIMARY KEY)
+
+statement ok
+CREATE TABLE b (id INT PRIMARY KEY)
+
+query TT colnames
+SHOW TABLES FROM test
+----
+Table
+a
+b
+
+statement ok
+DROP TABLE a
+
+query T colnames
+SHOW TABLES FROM test
+----
+Table
+b
+
```

Comment:
Can you add some rows into table a. Run select to ensure the rows are present. Then drop it as you've done, and recreate it. Run select on the table to ensure that it's empty. This will check that the truncate was indeed called. Thanks